tgui/packages/tgui/interfaces/GasMixer/MixInfo.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author Garash2k (https://github.com/garash2k)
+ * @license MIT
+ */
+
+import type { airInfo } from './types';
+
+interface MixInfoProps {
+  mix: airInfo;
+}
+export const MixInfo = (props: MixInfoProps) => {
+  const { mix } = props;
+  return (
+    <>
+      {mix.gasses
+        .filter((gas) => gas.Ratio > 0)
+        .sort((gas) => gas.Ratio)
+        .reverse()
```

Comment:
```suggestion
        .sort((gas) => gas.Ratio)
        .reverse()
```
Just sort in reverse order? Also `sort` takes a function with two arguments, not one. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort